FireboltNETSDK/FireboltClient.cs

```diff
@@ -226,6 +238,44 @@ protected async Task<T> SendAsync<T>(HttpMethod method, string uri, HttpContent?
 
         try
         {
+            string updateParametersHeader = "Firebolt-Update-Parameters";
+            if (response.Headers.Contains(updateParametersHeader))
+            {
+                string? newDatabase = null;
+                string? newEngine = null;
+                bool reconnect = false;
+                foreach (string[] kv in response.Headers.GetValues(updateParametersHeader).Select(p => p.Split('=', 2, StringSplitOptions.TrimEntries)))
+                {
+                    if (kv.Length == 1 || "".Equals(kv[1]))
+                    {
+                        connectionParameters.Remove(kv[0]);
+                    }
+                    else
+                    {
+                        connectionParameters[kv[0]] = kv[1];
+                        switch (kv[0])
+                        {
+                            case "database":
+                                newDatabase = kv[1];
+                                reconnect = true;
+                                break;
+                            case "engine":
+                                newEngine = kv[1];
+                                reconnect = true;
+                                break;
```

Comment:
Exactly, but this is a part of the document, so I have added this. It was not so difficult and I do not think that it bothers to much. As far as I understand support of `user engine` command will be requested soon.

Reply:
The document was for the services team to agree on the design. As we saw with USE DATABASE those designs can still change. I wouldn't jump the gun and implement this because there's a high change we'll just be doing the work twice. Moreover we have no way to test this functionality now so we're essentially adding untested code.

Reply:
removed